src/lucky_migrator/runner.cr

```diff
@@ -95,7 +100,7 @@ class LuckyMigrator::Runner
 
   def run_pending_migrations
     prepare_for_migration do
-      pending_migrations.each &.new.up
+      pending_migrations.each &.new.up(!@quiet)
```

Comment:
I may be misreading this PR but it looks like the "create" command is silenced, but the running migrations command is still putting a response

Reply:
The reason is because the `up` method takes a `log` param. So it should actually be `&.new.up(log: !@quiet)` which I think makes more sense.